utils/build.config

```diff
@@ -3,12 +3,11 @@ component=daos
 
 [commit_versions]
 ARGOBOTS = 89507c1f8cfec4e918e8b9861e41b4e9cef71461
-CART = 369b3686d68b784ecf6f66b675554b87c50dca56
+CART = v4.6.0
```

Comment:
there has been now update to cart master past 4.6.0 which fixes some of 'no route to host' issues reported in 4 or 5 different blocker tickets. Could we perhaps update to that version instead?

Reply:
Done in 049c4b3.